### ISSUE TITLE
Failed jobs do not appear in argus

### ISSUE THREAD

**Issue by ShlomiBalalis** [scylladb/argus#156]
http://13.48.103.68/workspace?state=WyIzZDA0NTRhNC0yMTViLTQ4NGUtODEzNy1hYTc5MzI5M2JkY2YiLCI2OTQ5MjY5ZS1kOTliLTQ0ZTYtYWYwNC1iMzUwM2Q2YjE0MDUiLCIxMzQ1Y2M5OS0wNWY4LTQ5NTUtYjdjYi1mMjBmOTAzODFiM2MiLCI2ODk3MDUxYS1kYjY3LTRmYmMtODZiNS05OGU2ZjUxYTc5ZWUiLCI3ZDllNDViZi00OWQ1LTQ3YWItYWFkMi01NTllZGZjMDc3MmYiLCJjOTJkMjkxMC03MTNjLTQ3MTgtODUzYy0wMDA5NzQ4M2UyZTIiXQ

The following runs and failed a few seconds into the `Run SCT Test` stage, which seem to have caused them to disappear
* scylla-cloud-longevity-100gb-48h-test#122
* scylla-cloud-operations-24h-aws#58
* scylla-cloud-longevity-small-data-set-1h-aws#515-517
* scylla-cloud-longevity-cdc-100gb-4h#9
* cloud-longevity-small-data-set-1h-gcp#252-253
* sct-siren-compatibility-checker#128

From the ones I checked so far, their logs weren't even collected for those runs, but it is still important that they will appear in argus so I know they ran in the first place

**Comment by rayakurl** [comment]
@k0machi , @roydahan - FYI

**Comment by roydahan** [comment]
@ShlomiBalalis do you mean that the jobs failed so early in the process that they didn't report to Argus yet?
If so, I think it's a known issue, and you can close it as such for now.

**Comment by k0machi** [comment]
Yep, it's a known issue, currently client mostly runs in that stage, so any early failures are not caught. We need to separate some client operations into their own stages to protect against that and protect against "stuck jobs".
